backend/api/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 import com.virtuslab.gitmachete.buildsrc.*
 
 dependencies {
+  implementation(project(":binding"))
```

Comment:
Hmmm an `api` depending on `binding`... looks fishy, lemme see the actual use...

Reply:
Okay, makes sense 🤔